src/pages/VacancyDetail.tsx

```diff
@@ -2,95 +2,106 @@ import earnData from "../assets/data/earn.json";
 import defaultInfo from "../assets/data/earnDefaultInfo.json";
 
 const VacancyDetail = ({ match }) => {
-  const currentVacancy = earnData.assignments.find(x => x.assignmentUrl === match.url);
+  const currentVacancy = earnData.assignments.find((x) => x.assignmentUrl === match.url);
```

Comment:
beide versies kunnen hier volgens mij maar als je hem liever in functie brackets hebt priem (:

Reply:
> beide versies kunnen hier volgens mij maar als je hem liever in functie brackets hebt priem (:

Denk dat m'n formatter dat heeft gedaan ^^